docs/sources/tempo/operations/grafana_datasource.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Grafana Tempo Datasource
+description: Use the Tempo Operator to deploy Tempo and use it as a data source with Grafana
+aliases:
+ - /docs/tempo/latest/gateway
+ - /docs/tempo/operations/gateway
+weight: 15
+---
+
+# Grafana Tempo Datasource
+
+Using the instructions on this page, you can configure the `TempoStack` to send data to Grafana and configure the Tempo data source.  
+
+You can choose to either use Tempo Operator's gateway or not: 
+
+* If the `TempoStack` is deployed using the gateway, you'll need to provide authentication information  to Grafana, along with the URL of the tenant it wants to access.
```

Comment:
In this phrase ("along with the URL of the tenant it wants to access"), "it" refers to tenant (the noun that precedes it). Do you mean tenant or the gateway? If the latter is correct, we should change "it" to gateway.

Reply:
none of them, I would like to express that you need to provide the URL of the tenant  you want to see the traces.

Reply:
Rephrased to make it more clear, let me know what do you think.